Buffer type is deprecated in Node

### Description
`Buffer` type is deprecated in Node (See https://sindresorhus.com/blog/goodbye-nodejs-buffer). Provoking the following error:
```
✘ [ERROR] TS2591: Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node` and then add 'node' to the types field in your tsconfig. [plugin angular-compiler]

    node_modules/fast-xml-parser/src/fxp.d.ts:409:26:
      409 │   parse(xmlData: string | Buffer ,validationOptions?: validationO...
```

### Input / Context
Consume the library from a TS project.

### Proposed Fix
Replace the `Buffer` type in file `fast-xml-parser/src/fxp.d.ts:409:26`  to be its super type: `Uint8Array`.

**Would you like to work on this issue?**
- [x] Yes. I will provide a PR for it.

--- a/src/fxp.d.ts
+++ b/src/fxp.d.ts
@@ -406,7 +406,7 @@ export type ValidationError = {
 
 export class XMLParser {
   constructor(options?: X2jOptions);
-  parse(xmlData: string | Buffer ,validationOptions?: validationOptions | boolean): any;
+  parse(xmlData: string | Uint8Array, validationOptions?: validationOptions | boolean): any;
   /**
    * Add Entity which is not by default supported by this library
    * @param entityIdentifier {string} Eg: 'ent' for &ent;

